<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-KGLKBPSV" failures="1" tests="3" name="FB-Test" time="11.12" errors="0" timestamp="2025-03-11T23:18:52 IST">
  <testcase classname="com.listenerz.ListnerzTest" name="fbTestFail" time="0.227">
    <skipped/>
  </testcase> <!-- fbTestFail -->
  <testcase classname="com.listenerz.ListnerzTest" name="fbTestFail" time="0.249">
    <skipped/>
  </testcase> <!-- fbTestFail -->
  <testcase classname="com.listenerz.ListnerzTest" name="fbTestFail" time="0.11">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.listenerz.ListnerzTest.fbTestFail(ListnerzTest.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- fbTestFail -->
  <testcase classname="com.listenerz.ListnerzTest" name="fbTestPass" time="0.0">
    <ignored/>
  </testcase> <!-- fbTestPass -->
</testsuite> <!-- FB-Test -->
